src/main/resources/logback.xml

```diff
@@ -11,16 +11,26 @@
         </encoder>
     </appender>
 
-
     <appender name="ASYNC" class="ch.qos.logback.classic.AsyncAppender">
+        <filter class="ch.qos.logback.classic.filter.ThresholdFilter">
+            <level>${ASYNC_LOGGING_LEVEL:-INFO}</level>
+        </filter>
         <appender-ref ref="STDOUT" />
     </appender>
 
+    <appender name="JSON" class="ch.qos.logback.core.ConsoleAppender">
```

Comment:
This logger should block when a log call is requested, as opposed to the AsyncAppender

There are also [logstash specific ones](https://github.com/logstash/logstash-logback-encoder#async-appenders)

Reply:
this appender seems to be not working, why it should block? it should log like current one

Reply:
logback's regular appenders simply write to disk, which means that they block on a IO operation, blocking the current thread until it's done.

Akka docs advice against that because the actor system threads are precious (any thread can be blocking thousands of actors messages on queue)...
so they suggest using the [async appender](http://logback.qos.ch/manual/appenders.html#AsyncAppender) on top of the regular one, using a custom dispatcher to log messages.

Reply:
> This logger should block when a log call is requested, as opposed to the AsyncAppender

I actually meant that this console appender would block, not "should"

Reply:
ok, I updated logback.xml